docs/reference/NuGet-Client-SDK.md

```diff
@@ -33,13 +33,28 @@ You can find the source code for these packages in the [NuGet/NuGet.Client](http
 > For documentation on the NuGet server protocol, please refer to the [NuGet Server API](~/api/overview.md).
 
 ## Support policy
+The most recent version of NuGet Client SDK is fully supported and can be relied on for bug fixes, updates, and enhancements.
+
+The recommendation is to use the latest versions of NuGet Client SDK packages, and to examine your project for dependencies on deprecated NuGet Client SDK packages.
+
+Out-of-support NuGet Client SDK packages will be [deprecated on nuget.org](../nuget-org/Deprecate-packages.md#client-experience-for-deprecated-packages).
+
+### Patch Releases
```

Comment:
Does this paragraph need to be moved to line 51?

I think the API stability stuff should be in the support policy.

Reply:
I see this was resolved, but I didn't see the paragraph moved, did you miss a push?